pilot/cmd/pilot-discovery/main.go

```diff
@@ -41,6 +43,11 @@ var (
 		KeepaliveOptions: keepalive.DefaultOption(),
 	}
 
+	mcpAddrs = env.RegisterStringVar(
+		"MCPADDRS",
```

Comment:
This name isn't very clear ... this is specifically for synthetic service entries, right?

Reply:
Yeah, it is currently only used for SSEs. Open to suggestion if you have one :)

Reply:
Maybe something like `SSE_MCP_ADDRESSES`?